addons/website_sale_collect/__manifest__.py

```diff
@@ -16,6 +16,7 @@
 
         'views/delivery_carrier_views.xml',
         'views/delivery_form_templates.xml',
+        'views/delivery_pick_up_in_store_views.xml',
```

Comment:
Could you please rename the file?
```suggestion
        'views/stock_picking_views.xml',
```